### PR TITLE
Add in gajira integration

### PR DESCRIPTION
**Pull request by TawneeOwl**
## What does this pull request do?

Adds in jira ticket integration for the new access service

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

**Review by Pheonnexx — APPROVED** [review]
LGTM

File: .github/workflows/gajira.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Gajira
```
Comment (BenMillar-CGI) [inline .github/workflows/gajira.yml:1]: Would it be possible to name this something more descriptive? i.e. "Raise Jira Issue"

**Review by BenMillar-CGI — APPROVED** [review]
Looks good to me, thanks for this!

**Review by BenMillar-CGI — CHANGES_REQUESTED** [review]
I have left two comments around naming, feel free to adjust these or dismiss as you wish.

**Review by BenMillar-CGI — APPROVED** [review]
LGTM, thank you 👍

**Pull request merged**